generate/input/descriptor.json

```diff
@@ -1667,6 +1667,20 @@
           "return": {
             "isResultOrError": true
           }
+        },
+        "git_graph_reachable_from_any": {
```

Comment:
Can we get a test for this?

Reply:
turns out this function was totally broken. wouldn't have known without trying to test it :sweat_smile: